code/API_definitions/BYON-Notification-Channel.yaml

```diff
@@ -117,12 +130,12 @@ paths:
             schema:
               $ref: '#/components/schemas/NotificationChannel'
```

Comment:
Myabe it is not correct to include `ChannelId `in the Post method? Is it not returned by the platfform?

@deepakjaiswal1 @pradeepachar-mavenir @rartych

Reply:
@TEF-RicardoSerr  The NotificationChannel schema is utilized for both Requests and Responses and therefore includes a ChannelId. Please note that the ChannelId should not be set or provided by the client; it will be returned by the platform in the Response.